src/tests/ftest/aggregation/basic.yaml

```diff
@@ -9,6 +9,7 @@ server_config:
     0:
       log_mask: INFO
       storage: auto
+  system_ram_reserved: 64
```

Comment:
Why does just this test need this? Wouldn't others need the same?

Reply:
I'm not sure why the usage of memory on a single host was so high in the specific instance of the test running in CI, I'm not sure whether that's to do with the test itself. As mentioned in one of the above comments it's not expected that a host running the test would have other applications using so much memory during the test so that's why such a large buffer is required to take this odd scenario into account. As observed in the ticket comment: https://daosio.atlassian.net/browse/DAOS-14203?focusedCommentId=119749 it's strange that one host is reporting 163 available RAM and the other 123 (which causes the failure) when they both have the same amount of installed RAM. There is one other test yaml that has the value set to 64, think it's one of the DMG tests. This scenario where 64/187 gib of RAM is being used elsewhere on the host when trying to run DAOS wouldn't be realistically expected in production, hence why the check is failing and why adjustment of the tunable is required.